config.mk

```diff
@@ -92,9 +92,10 @@ else ifeq ($(OS),Linux)
 
     ifeq ($(ARCH),x86_64)
       export ENABLEFMA4=1
-      export ENABLEAVX512F= $(shell expr `$(CC) -dumpversion | sed 's/\..*//g'` '>=' 5)
+      export ENABLEAVX512F= $(shell expr `$(CC) -dumpversion | sed 's/\..*//g'` '>=' 6)
       export ENABLEFLOAT80= $(shell expr `$(CC) -dumpversion | sed 's/\..*//g'` '>=' 5)
-      export ENABLEFLOAT128= $(shell expr `$(CC) -dumpversion | sed 's/\..*//g'` '>=' 5)
+      export ENABLEFLOAT128= $(shell expr `$(CC) -dumpversion | sed 's/\..*//g'` '>=' 7)
```

Comment:
Could you explain why you set the minimum version number of gcc for enableing float128 support to 7? It should be available in version 5. Is it a workaround of some bug?